test/test_tensor_creation_ops.py

```diff
@@ -732,7 +732,7 @@ def test_linspace_vs_numpy(self, device, dtype):
 
     @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
     @precisionOverride({torch.float: 1e-6, torch.double: 1e-10})
-    @dtypes(torch.float, torch.double)
+    @dtypes(*torch.testing.get_all_fp_dtypes(include_half=False, include_bfloat16=False))
```

Comment:
This is OK but you're not testing bfloat16 in this PR. You may need to add a separate, "easier" tests that compares bfloat16 vs. NumPy's logspace in fp32.

Reply:
Similar to `linspace`, I think this test simply test its consistency with NumPy. The modification of the decorator of `test_logspace` below adds bfloat16 into the test.

Reply:
OK; that seems reasonable.